shell/server/feature-key.js

```diff
@@ -17,20 +17,25 @@
 const Capnp = Npm.require("capnp");
 const FeatureKey = Capnp.importSystem("sandstorm/feature-key.capnp").FeatureKey;
 
-// These byte-packing tricks that are so convenient in non-memory-safe languages are a
-// bit of a pain in memory-safe languages.  Argh.  At least I have bignums on the server.
+// We don't currently have a way of reading the raw bytes of a capnp struct in Javascript the
+// way we do in C++. Awkwardly, what we end up with is four 64-bit ints that have been stringified
+// since JS can't actually deal with 64-bit ints. We have to laboriously reconstruct the underlying
+// bytes using Bignum. Don't forget that everything is little-endian, as DJBeesus intended.
 const bits0 = Bignum(FeatureKey.signingKey.key0);
 const bits1 = Bignum(FeatureKey.signingKey.key1);
 const bits2 = Bignum(FeatureKey.signingKey.key2);
 const bits3 = Bignum(FeatureKey.signingKey.key3);
-const signingKey = bits0.shiftLeft(64 * 3)
-    .add(bits1.shiftLeft(64 * 2))
-    .add(bits2.shiftLeft(64))
-    .add(bits3)
-    .toBuffer();
+const signingKey = bits0
+    .add(bits1.shiftLeft(64 * 1))
+    .add(bits2.shiftLeft(64 * 2))
+    .add(bits3.shiftLeft(64 * 3))
+    .toBuffer({ endian: "little", size: 32 });
+
+const isTesting = Meteor.settings && Meteor.settings.public &&
+                  Meteor.settings.public.isTesting;
 
 // Export for use in Meteor method in admin.js
-verifyFeatureKeySignature = function (buf) {
+function verifyFeatureKeySignature(buf) {
```

Comment:
Actually binding a name with `function` has similar semantics to `var`, so won't be visible in other files. Arguably, perhaps, we should move to a style of declaring all functions as `const f = (...) =>` for the same reasons we've banned `var`, but currently it looks like we still declare many things using `function` so I'll leave it for a later mass cleanup.